cmd/juno/juno.go

```diff
@@ -122,7 +122,7 @@ const (
 	defaultCNL2ChainID              = ""
 	defaultCNCoreContractAddressStr = ""
 	defaultCallMaxSteps             = 4_000_000
-	defaultGwTimeout                = 5 * time.Second
+	defaultGwTimeout                = "5s,"
```

Comment:
So we are not dynamic by default 🤔 . I was thinking we should. Let's be dynamic for a while